core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerEventForwarder.java

```diff
@@ -0,0 +1,98 @@
+package com.novoda.noplayer.exoplayer;
+
+import android.view.Surface;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.decoder.DecoderCounters;
+import com.google.android.exoplayer2.upstream.DataSpec;
+
+import java.io.IOException;
+
+abstract class ExoPlayerEventForwarder implements Forwarder {
```

Comment:
All `Forwarder`s were collapsed into one interface to make it easier to understand and forward events between the `ExoPlayerTwoFacade` and the `ExoPlayerTwoImpl`. There are children forwarders of this that know only the methods they are interested in that perform the conversion from `ExoPlayer` to `NoPlayer` callbacks.